tests/phpunit/tests/html-api/wpHtmlProcessor.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Unit tests covering WP_HTML_Processor functionality.
+ *
+ * @package WordPress
+ * @subpackage HTML-API
+ */
+
+/**
+ * @group html-api
+ *
+ * @coversDefaultClass WP_HTML_Processor
+ */
+class Tests_HtmlApi_wpHtmlProcessor extends WP_UnitTestCase {
```

Comment:
```suggestion
class Tests_HtmlApi_WpHtmlProcessor extends WP_UnitTestCase {
```
See [this ticket](https://core.trac.wordpress.org/ticket/56846) regarding `Wp` instead of `wp` for test class names.

Reply:
why don't we handle these kinds of style changes in one sweep instead of letting them slide for ages? the Tag Processor merged _after_ that ticket but at some point someone told me, I think, that it whould be lowercase there, so I copied that here.

Reply:
instead of "fixing" this here and introducing a very odd quirk between the existing tests and the new ones, I'd prefer we resolve this outside of this PR. if you want to rename them and merge the changes to the existing tests, I'll rebase this work on top of that and then we can separate _introducing new functionality_ from _changing existing style preferences_

Reply:
No worries! We can handle these in one sweep as you say.

Reply:
followed up in https://github.com/WordPress/wordpress-develop/pull/4897